src/gui/dialogs/window_dlg_preheat.cpp

```diff
@@ -14,6 +14,13 @@
 
 /*****************************************************************************/
 //NsPreheat::I_MI_Filament
+NsPreheat::I_MI_Filament::I_MI_Filament(string_view_utf8 name, unsigned t_noz, unsigned t_bed)
+    : WI_INFO_t(name, nullptr, is_enabled_t::yes, is_hidden_t::no) {
+    char buff[sizeof("999/999 ")];
+    snprintf(buff, sizeof(buff), "%3u/%3u ", t_noz, t_bed); // extra space at the end is intended
```

Comment:
Could you also mention the reason for the extra space? I assume it's to shift the numbers to the left a bit, but why is it needed in this particular case?